grails-app/domain/org/broadinstitute/orsp/IssueExtraProperty.groovy

```diff
@@ -40,6 +42,10 @@ class IssueExtraProperty {
     public static final String DATA_SHARING_BROAD = "dataSharingBroad"
     public static final String DATA_SHARING_COMMENTS = "dataSharingComments"
     public static final String DATA_SHARING_NIH = "dataSharingNih"
+    public static final String COLLABORATORS = "collaborators"
+    public static final String PROJECT_MANAGER = "pm"
+    public static final String IRB_PROTOCOL_ID = "protocol"
```

Comment:
Note that we already have a `protocol` on line 12. This line isn't necessary and will actually be a conflict.